client/record/cli/query.go

```diff
@@ -0,0 +1,87 @@
+package cli
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/wire"
+	"github.com/irisnet/irishub/client"
+	"github.com/irisnet/irishub/client/context"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+)
+
+func GetCmdQureyHash(cdc *wire.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tx [hash]",
+		Short: "Matches this txhash over all committed blocks",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// find the key to look up the account
+			hashHexStr := viper.GetString(FlagTxHash)
+
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+			output, err := queryTx(cdc, cliCtx, hashHexStr)
```

Comment:
We can use the 'func (cliCtx CLIContext) QueryStore' instead of introducing a new function to query and validate proof.

Reply:
Function GetCmdQureyHash is already updated in latest commit, and queryTx was removed.

Reply:
Could you please post the 'last commit' hash?